src/lib.rs

```diff
@@ -536,13 +717,13 @@ impl IntoRawFd for UnixStream {
 /// // close the listener socket
 /// drop(listener);
 /// ```
-pub struct UnixListener {
+pub struct UnixStreamListener {
```

Comment:
:(
o.k.

On Fri, May 27, 2016, 12:42 AM Steven Fackler <notifications@github.com>
wrote:

> In src/lib.rs
> <https://github.com/rust-lang-nursery/unix-socket/pull/25#discussion_r64857039>
> :
>
> > @@ -536,13 +717,13 @@ impl IntoRawFd for UnixStream {
> >  /// // close the listener socket
> >  /// drop(listener);
> >  /// ```
> > -pub struct UnixListener {
> > +pub struct UnixStreamListener {
>
> I think we should probably leave this as UnixListener. It's a bit of an
> unfortunate naming setup, but that's what the standard library setup has,
> and it's slated for stabilization.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-lang-nursery/unix-socket/pull/25/files/fcb59380b36d9182b7b83c1db74ef0c9f345674c#r64857039>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHlC24C1s3x9VvOxC7H9USyEkFBwTLXks5qFnZOgaJpZM4IoKe0>
> .
>